examples/MCP/simple_calculator_mcp/README.md

```
@@ -82,15 +82,14 @@ You can run the simple calculator workflow using Remote MCP tools. In this case,
 You can publish the simple calculator tools via MCP using the `nat mcp` command. Details are provided in the [MCP Server Guide](../../../docs/source/workflows/mcp/mcp-server.md).
 
 ## Configuration Examples
-
-| Configuration File | MCP Server Type | Available Tools |
-|-------------------|-----------------|-----------------|
-| `config-mcp-date.yml` | Date Server | Current time, date formatting |
-| `config-mcp-math.yml` | Math Server | Advanced mathematical operations |
-| `config-combined.yml` | Multiple Servers | Combined demonstration |
+| Configuration File | MCP Server Type | Transport | Available Tools |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)